modules/standard/JSON.chpl

```diff
@@ -1185,7 +1185,12 @@ module JSON {
   proc fromJson(jsonString: string, type loadAs): loadAs throws {
```

Comment:
Could you add a warning to the docs about this requirement? Or maybe a ``compilerWarning`` guarded by ``!isDefaultInitializable(loadAs)``?

Reply:
The warning is added in the current version.